pandas/util/_exceptions.py

```diff
@@ -11,6 +11,9 @@
     from collections.abc import Generator
     from types import FrameType
 
+Pandas40DeprecationWarning = DeprecationWarning
+Pandas50DeprecationWarning = DeprecationWarning
```

Comment:
Considering pandas only enforces deprecations on major releases, this sounds more like pandas v40 rather than v4.0, should I rename this to be `Pandas4DeprecationWarning` instead?

Reply:
Both are fine with me.